test/integration/compute_profile_js_test.rb

```diff
@@ -51,6 +51,7 @@ class ComputeProfileJSTest < IntegrationTestWithJavascript
   test "create compute profile" do
     visit compute_profiles_path()
     click_on("Create Compute Profile")
+    page.driver.browser.file_detector = nil if page.driver.browser.respond_to?(:file_detector=)
```

Comment:
What is the file_detector?

Reply:
This is a workaround for a [bug in selenium](https://stackoverflow.com/questions/70441796/selenium-webdriver-for-aws-device-farm-error-when-sending-period-keystroke-t) which was fixed the same way in [foreman_puppet](https://projects.theforeman.org/issues/37251)

Reply:
That's nasty. Is there a bug in report in capybara we can link to?

How about we create a helper like `work_around_selenium_file_detector_bug` in `test/integration_test_helper.rb` that does this? Then other plugins can also easily use this and we can easily identify where it's needed.

Reply:
> That's nasty. Is there a bug in report in capybara we can link to?
I couldn't find one. Though from the so post I'm not sure if it is a bug/feature in selenium or capybara.

> How about we create a helper like `work_around_selenium_file_detector_bug` in `test/integration_test_helper.rb` that does this? Then other plugins can also easily use this and we can easily identify where it's needed.
Makes sense.